integration/spark2/src/main/scala/org/apache/spark/sql/CarbonDatasourceHadoopRelation.scala

```diff
@@ -48,7 +48,8 @@ case class CarbonDatasourceHadoopRelation(
     isSubquery: ArrayBuffer[Boolean] = new ArrayBuffer[Boolean]())
   extends BaseRelation with InsertableRelation {
 
-  lazy val identifier: AbsoluteTableIdentifier = AbsoluteTableIdentifier.fromTablePath(paths.head)
+  lazy val identifier: AbsoluteTableIdentifier = AbsoluteTableIdentifier.from(paths.head,
+    parameters("dbname"), parameters("tablename"))
```

Comment:
Check the camel case of parameters, please debug and verify oce

Reply:
the map parameters is case insensitive